packages/frontend/src/components/sign/SignTransferMultipleAccounts.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { Translate } from 'react-localize-redux';
+import styled from 'styled-components';
+
+import ArrowUpImage from '../../images/icon-arrow-up-green.svg';
+import DangerIcon from '../../images/icon-danger-circle.svg';
+import FormButton from '../common/FormButton';
+import Container from '../common/styled/Container.css';
+import SafeTranslate from '../SafeTranslate';
+
+const CustomContainer = styled(Container)`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    color: #25282a;
+    text-align: center;
+
+    .title {
+        margin-top: 23px;
+
+        h2 {
+            font-weight: 900;
+            font-size: 22px;
+            color: #24272a;
+        }
+    }
+
+    && .text {
+        color: #72727a;
+        margin-top: 24px;
+    }
+
+    .buttons {
+        display: flex;
+        width: 100%;
+        margin-top: 40px;
+
+        button {
+            flex: 1;
+
+            &:last-of-type {
+                margin-left: 30px;
+
+                @media (min-width: 768px) {
+                    margin-left: 50px;
+                }
+            }
+        }
+    }
+
+    .fees {
```

Comment:
Maybe leftover? Don't see any `html` for this style

Reply:
Thanks for catching 😄